test/any/FetchWebIDLFile-test.js

```diff
@@ -0,0 +1,156 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+describe('WebIDLFile', function() {
```

Comment:
Just realized this isn't the name of an actual class. Let's do a description like `Web IDL file classes`.